web/packages/teleterm/src/ui/services/deepLinks/deepLinksService.ts

```diff
@@ -88,17 +90,52 @@ export class DeepLinksService {
         break;
       }
       case '/authenticate_web_device': {
-        // TODO (avatus): add authenticate device web confirmation dialog
-        // this case is a stub and will not be reached
+        const id = result.url.searchParams.id;
+        const token = result.url.searchParams.token;
+        await this.askAuthorizeDeviceTrust(result.url, id, token);
```

Comment:
`id` and `token` are aready in `result.url`. What do you think about making `askAuthorizeDeviceTrust` accept `AuthenticateWebDeviceDeepURL` instead of `DeepURL` and then just plucking id and token out of search params inside that method?